docs/Custom_Device_Types.md

```diff
@@ -104,19 +104,24 @@ The *Close* case executes after the PCL has finished executing. You should close
 
 ### Inline-Async Hardware Interface
 
-By checking the **Use Inline-Async API** for the *Inline HW Interface* Custom Device Execution Mode in the niveristand-custom-device-wizard, an inline-async custom device template will be generated.
+You can generate an inline-async custom device template from the niveristand-custom-device-wizard. When creating a project, set the execution mode to **Inline HW Interface** and enable **Use Inline-Async API**.
 
 ![](images/Use_Inline_Async_API.jpg)
 
-By using the [Inline-Async-API](https://github.com/ni/niveristand-custom-device-development-tools/tree/main/inline-async-api) the inline-async template framework provides initializing/launching/cleanup of the asynchronous VIs, error handling/reporting, and data transfer between inline and asynchronous VIs.
+By using the [Inline-Async-API](https://github.com/ni/niveristand-custom-device-development-tools/tree/main/inline-async-api), the inline-async template framework can perform the following actions.
+* Initializing asynchronous VIs.
+* Launching asynchronous VIs.
+* Cleaning up asynchronous VIs.
+* Handling and reporting errors.
+* Transferring data between inline and asynchronous VIs.
 
-The RT Driver VI of an inline custom device can communicate channel data with VeriStand while launching an asynchronous loop(s) to handle nondeterministic operations.
+The RT Driver VI of an inline custom device can communicate channel data with VeriStand. While doing so, the VI can also launch an asynchronous loop(s) to handle nondeterministic operations.
 
 One example of a nondeterministic operation is writing data to a log file. The RT Driver VI of the inline custom device communicates with the asynchronous loop(s) using RT FIFOs.
 
 ![](images/Inline-Async_Engine_lvlib_RT_Driver.jpg)
 
-**Note:** Data must be consumed from the RT FIFOs at a fast enough rate or the mechanism will overflow.
+**Note:** The custom device must consume data from the RT FIFOs at a fast enough rate or it will overflow.
```

Comment:
@phutanu Is there a way to increase this rate?

Reply:
This will not fix the underling issue, even if we set the max rate size.
An inline-async custom device has two parts, inline and async. This parts need to send data back and fourth, depending on what the user wants to do. The inline part of the custom device will run in sync with the PCL, or at a decimation. And depending on how data is sent (from inline to async, or from async to inline), the users need to make sure the mechanism will not create data loss (async runs too slow, and RT FIFO fills-up, or other).